cirq-core/cirq/devices/thermal_noise_model.py

```diff
@@ -228,10 +230,38 @@ def __init__(
         _validate_rates(qubits, rate_dict)
         self.gate_durations_ns: dict[type, float] = gate_durations_ns
         self.rate_matrix_GHz: dict[cirq.Qid, np.ndarray] = rate_dict
+        self.heat_rate_GHz = heat_rate_GHz
+        self.cool_rate_GHz = cool_rate_GHz
+        self.dephase_rate_GHz = dephase_rate_GHz
         self.require_physical_tag: bool = require_physical_tag
         self.skip_measurements: bool = skip_measurements
         self._prepend = prepend
 
+    def _value_equality_values_(self):
+        gate_times = tuple(sorted((k, v) for k, v in self.gate_durations_ns.items()))
```

Comment:
`gate_durations_ns` can be used as is, no need for conversion here.

Reply:
> `gate_durations_ns` can be used as is, no need for conversion here.

Ack.  In order to support hashing, `_value_equality_values_` must return a hashable object.  Improved sorting key in 0a1332d780a300be4395f2120242f4ffc36f1d0e for better order reproducibility.